React/React.xcodeproj/project.pbxproj

```diff
@@ -3951,7 +3950,6 @@
 				6B857F421EC51FCF00A9D063 /* libcxxreact.a */,
 				1882027C1EF48B7E00C9B354 /* libdouble-conversion.a */,
 				188202A21EF48CF700C9B354 /* libthird-party.a */,
-				18F3BA0E2142F4DB00AD247D /* RCTWebView.m */,
```

Comment:
The RCTWebView.m is not actually in the repo anymore, so it can't be building into the lib -- which is good.
All five of these "Recovered References" could be removed.

Reply:
Actually we get build failures when we remove the libs from the Recovered References. Maybe they can to get moved elsewhere for clarity, but this is also non-impactful and gets fixed in 0.61 so I don't think there's value in spending too much more time on it.